packages/experiments/src/test/index.js

```diff
@@ -37,18 +37,6 @@ describe( '__dangerousOptInToUnstableAPIsOnlyForCoreModules', () => {
 			/This feature is only for JavaScript modules shipped with WordPress core/
 		);
 	} );
-	it( 'Should not register the same module twice', () => {
```

Comment:
Why is this test not necessary anymore?

Reply:
`IS_GUTENBERG_PLUGIN` is set when running unit tests so it fails because we've disabled the error in this case. We could mock unset `IS_GUTENBERG_PLUGIN`, but I'm not sure if that's right... the unit tests are supposed to test the _plugin_, right?

Reply:
Right, it makes sense. If I understand correctly, then, I guess we can bring the unit test back if/when we use the `ALLOW_EXPERIMENTS_REREGISTRATION` flag instead ?

Reply:
Now that we're using a different env variable we can restore this test, which I've done.